DNN Platform/Library/Services/Installer/Util.cs

```diff
@@ -616,12 +616,6 @@ private static string ValidateNode(string propValue, bool isRequired, Logger log
         public static string CONFIG_Committed = GetLocalizedString("CONFIG_Committed");
         public static string CONFIG_RolledBack = GetLocalizedString("CONFIG_RolledBack");
         public static string CONFIG_Updated = GetLocalizedString("CONFIG_Updated");
-        public static string DASHBOARD_ReadSuccess = GetLocalizedString("DASHBOARD_ReadSuccess");

```

Comment:
Let's also remove the resources from the resx file: https://github.com/dnnsoftware/Dnn.Platform/blob/f5240f8148ad6bdae497fcba1964f49fd98cd756/DNN%20Platform/Website/App_GlobalResources/SharedResources.resx#L1407-L1424

Reply:
And this one: https://github.com/dnnsoftware/Dnn.Platform/blob/80e48fcde4694d70960253d401a8fcd6851608eb/Dnn.AdminExperience/Dnn.PersonaBar.Extensions/admin/personaBar/Dnn.Extensions/App_LocalResources/Extensions.resx#L252-L254

Reply:
Just pushed a commit for this one